frontend/docs/admin-guide/install.md

```diff
@@ -969,6 +969,232 @@ policy. From the IAM Service page click "Policies" on the left hand side.
 13. After selecting the 3 policies click next and enter a name for the role. You can name the role whatever you'd like. Optionally add a description and tags and then click "Create role"
 14. Once the role has been created record the role ARN as we'll need to use it later.
 
+#### System Role
+
+1. The example below assumes you've created the permission boundary policy above in the [App Role](App Role) section and named the policy `mlspace-project-user-permission-boundary`. If you've named it something different
+you'll need to update the `"iam:PermissionsBoundary` condition in the `iam:CreateRole` statement below.
+2. After the permission boundary policy has been created you are now ready to create the application
```

Comment:
I think this should say "system policy", but also see my note on that they can re-use the app policy if they desire.

Reply:
No they'll need the permission boundary policy like they have for the app role to ensure the system policy/role never has permissions outside the permission boundary.

Reply:
I think you may have misunderstood me here. This is the "System Role" section and in this bullet it says, "you are now ready to create the **application** policy".

Shouldn't this say "system policy"?